src/test/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/PopulateUpdateSubmissionTest.java

```diff
@@ -0,0 +1,210 @@
+
+  final String ANY_BOS_OR_MOS_ADDED_OR_CEASED_JSON = "\"anyBOsOrMOsAddedOrCeased\":false,\"";
+  private final EntityNameDto entityNameDto = EntityNameMock.getEntityNameDto();
+  private final EntityDto entityDto = EntityMock.getEntityDto();
+  private final PresenterDto presenterDto = PresenterMock.getPresenterDto();
+  private final BeneficialOwnersStatementType beneficialOwnersStatement =
+      BeneficialOwnersStatementType.NONE_IDENTIFIED;
+  private final DueDiligenceDto dueDiligenceDto = DueDiligenceMock.getDueDiligenceDto();
+  private final UpdateDto updateDto = UpdateMock.getUpdateDto();
+  private final List<BeneficialOwnerIndividualDto> beneficialOwnerIndividualDtoList =
+      new ArrayList<>();
+  private final List<BeneficialOwnerCorporateDto> beneficialOwnerCorporateDtoList =
+      new ArrayList<>();
+  private final List<BeneficialOwnerGovernmentOrPublicAuthorityDto>
+      beneficialOwnerGovernmentOrPublicAuthorityDtoList = new ArrayList<>();
+  private final List<ManagingOfficerIndividualDto> managingOfficerIndividualDtoList =
+      new ArrayList<>();
+  private final List<ManagingOfficerCorporateDto> managingOfficerCorporateDtoList =
+      new ArrayList<>();
+  private final List<TrustDataDto> trustDataDtoList = new ArrayList<>();
+
+  String output = "";
+  @InjectMocks private OverseasEntitySubmissionDtoValidator overseasEntitySubmissionDtoValidator;
+  @Mock private EntityNameValidator entityNameValidator;
+  @Mock private EntityDtoValidator entityDtoValidator;
+  @Mock private PresenterDtoValidator presenterDtoValidator;
+  @Mock private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+  @Mock private DueDiligenceDataBlockValidator dueDiligenceDataBlockValidator;
+  @Mock private OwnersAndOfficersDataBlockValidator ownersAndOfficersDataBlockValidator;
+  @Mock private TrustDetailsValidator trustDetailsValidator;
+  @Mock private TrustIndividualValidator trustIndividualValidator;
+  @Mock private HistoricalBeneficialOwnerValidator historicalBeneficialOwnerValidator;
+  @Mock private TrustCorporateValidator trustCorporateValidator;
+
+  {
+    beneficialOwnerIndividualDtoList.add(
+        BeneficialOwnerAllFieldsMock.getBeneficialOwnerIndividualDto());
+  }
+
+  {
+    beneficialOwnerCorporateDtoList.add(
+        BeneficialOwnerAllFieldsMock.getBeneficialOwnerCorporateDto());
+  }
+
+  {
+    beneficialOwnerGovernmentOrPublicAuthorityDtoList.add(
+        BeneficialOwnerAllFieldsMock.getBeneficialOwnerGovernmentOrPublicAuthorityDto());
+  }
+
+  {
+    managingOfficerIndividualDtoList.add(ManagingOfficerMock.getManagingOfficerIndividualDto());
+  }
+
+  {
+    managingOfficerCorporateDtoList.add(ManagingOfficerMock.getManagingOfficerCorporateDto());
+  }
+
+  {
+    trustDataDtoList.add(TrustMock.getTrustDataDto());
+  }
+
+  @Test
+  void testPopulateUpdateSubmissionJSON() throws Exception {
+    output = getJsonOutput();
```

Comment:
It's not clear what exactly we are testing here. Usually we would create an instance of the class and then call the method we want to test. We'd mock the data being passed to the method and then verify the response is what we expect. Hiding all of this inside of 'getJsonOutput' is not helpful at all